qiskit_dynamics/pulse/pulse_to_signals.py

```diff
@@ -188,14 +198,15 @@ def get_signals(self, schedule: Schedule) -> List[DiscreteSignal]:
             if isinstance(inst, ShiftPhase):
                 phases[chan] += inst.phase
 
+            if isinstance(inst, SetPhase):
```

Comment:
Is this related to the warning?

Reply:
This is not related to the warning: I saw that the `SetPhase` and `ShiftPhase` handling were not grouped together, so I moved this block up.